FND/Organizations/FormalOrganizations.rdf

```diff
@@ -93,12 +173,77 @@
 	</owl:Class>
 	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;employs">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;playsActiveRoleThatDirectlyAffects"/>
+		<rdfs:label>employs</rdfs:label>
+		<rdfs:range rdf:resource="&fibo-fnd-aap-ppl;Person"/>
+		<skos:definition>indicates someone that is employed by the legal person or organization</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployedParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-org-org;hasOrganizationMember"/>
+		<rdfs:label>has employed party</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employment"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employee"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployedIn"/>
+		<skos:definition>identifies the employee in an employment situation</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployee">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-pty-pty;actsOn"/>
+		<rdfs:label>has employee</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employer"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employee"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployeeOf"/>
+		<skos:definition>indicates an employee that is employed by the employer</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;hasEmployingParty">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-org-org;hasMembership"/>
+		<rdfs:label>has employing party</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;Employment"/>
+		<rdfs:range rdf:resource="&fibo-fnd-org-fm;Employer"/>
+		<owl:inverseOf rdf:resource="&fibo-fnd-org-fm;isEmployingParty"/>
+		<skos:definition>identifies employer in an employment situation</skos:definition>
+	</owl:ObjectProperty>
+	
 	<owl:ObjectProperty rdf:about="&fibo-fnd-org-fm;isDomiciledIn">
 		<rdfs:label>is domiciled in</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-fnd-org-fm;FormalOrganization"/>
 		<rdfs:range rdf:resource="&lcc-cr;GeopoliticalEntity"/>
 		<skos:definition>indicates a primary location where an entity conducts business, such as where its headquarters is located</skos:definition>
 		<fibo-fnd-utl-av:explanatoryNote>Corporate domicile refers to a place where a company&apos;s affairs are discharged. It is also typically the legal home of a corporation because the place is considered by law as the centre of corporate affairs. In cases where a business has incorporated in one location for convenience, such as for taxation, legal, or regulatory purposes, but operates primarily in one or more other locations, domicile in FIBO refers to the operational location(s) rather than legal location.  Many companies in the US have incorporated in the State of Delaware, for example, but do not have operational facilities in Delaware (or only have small offices there).</fibo-fnd-utl-av:explanatoryNote>
```

Comment:
centre->center for US

Reply:
That was an MB definition originally - I'll fix it.